packages/http-date/http-date.0.1/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "HTTP Datetime encoder/decoder"
+description: "RFC 9110 compliant HTTP datetime decoder and encoder"
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem"]
+license: "MPL-2.0"
+homepage: "https://github.com/bikallem/http-date"
+bug-reports: "https://github.com/bikallem/http-date/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "3.6" & >= "2.6"}
```

Comment:
```suggestion
  "dune" {>= "3.6"}
```
You can remove the dune dependency in the dune-project file and it will resolve this double bound